config/locales/en.yml

```diff
@@ -254,6 +254,13 @@ en:
         happen_next_heading: What happens next
         happen_next_text: "You will be sent the results of your client's financial assessment so you can confirm they are eligible for legal aid."
         back_button: "Back to your applications"
+    own_homes:
+      show:
+        h1-heading: Does your client own the home that they live in?
+        own_home:
```

Comment:
This section is the same as the citizen version starting on line 311. Is there a way of reusing that rather than defining everything twice? (Not a blocker...)

Reply:
If you look - the heading is different. "Does your client own the home that they live in?" rather than  "Do you own the home that you live in?".

Reply:
Following discussion, it was agreed that having all the translations for a provider page, with the provider section of the translation file was less likely to cause confusion, and therefore the current format should be kept.

Reply:
👍

Reply:
For the actual field labels, a 3rd option would be to use a `&` variable like use there:
https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/config/locales/en.yml#L47

```
    attributes:
      address: &address_attrs
        address_line_one: 'Building and street'
        address_line_two: ''
        city: 'Town or city'
        county: 'County'
        postcode: 'Postcode'
....
....
      applicants/address_form:
        <<: *address_attrs
```

Reply:
That's the best compromise

Reply:
However, my solution doesn't work on error, because on error the `show` template is rendered in the `update` action - kicking out the translation path.

I've reverted it all to use the default path the form file syntax.